specs.json

```diff
@@ -153,6 +153,14 @@
   "https://immersive-web.github.io/anchors/",
   "https://immersive-web.github.io/model-element/",
   "https://immersive-web.github.io/raw-camera-access/",
+  {
+    "url": "https://immersive-web.github.io/real-world-geometry/plane-detection.html",
```

Comment:
The IDL of that spec is incomplete (missing `[Exposed]` on the interface), which IIUC will block the IDL package release down the line. I'm curious if there is a better way to deal with this than just going through the cascade of releases.

Reply:
I do not have a fully satisfactory solution to propose today, either:
1. we put the spec in the monitor list for now since we know that the IDL is invalid; or
2. we create a patch; or
3. we wait until the IDL gets fixed

I'm surprised that we did not seem to have any open issue on the topic, even though we've been floating the idea to improve the curation process in that regard for some time. I created https://github.com/w3c/webref/issues/1055 to track this.

Another option could be to automate checks a bit more when we add a spec to browser-specs, e.g., by running Reffy and tests on the spec and prevent addition of the spec while checks fail. That seems nice in theory but more complex in practice. Also, that wouldn't solve the problem for specs that start to fail after some time.